rudder-core/src/main/scala/com/normation/rudder/migration/XmlFileFormatMigrationEntities.scala

```diff
@@ -38,154 +38,61 @@
 
 package com.normation.rudder.migration
 
-import java.sql.ResultSet
-import java.sql.Timestamp
-import java.util.Calendar
+import com.normation.rudder.db.DB
 
-import scala.xml.Elem
-import scala.xml.XML
+import org.joda.time.DateTime
+import com.normation.rudder.db.Doobie
+import scalaz._, Scalaz._
+import doobie.imports._
+import scalaz.concurrent.Task
 
-import org.springframework.jdbc.core.RowMapper
-import org.squeryl.KeyedEntity
-import org.squeryl.PrimitiveTypeMode._
-import org.squeryl.Schema
-import org.squeryl.annotations.Column
+class MigrationEventLogRepository(val db: Doobie) {
 
-import com.normation.rudder.repository.jdbc.SquerylConnectionProvider
+  import db._
```

Comment:
no, the convention is on the serialize object, DB.XXXX. And it's just a convention. 
Here, it's the name of the connection provider.